.circleci/config.yml

```diff
@@ -1,15 +1,15 @@
 version: 2.1
 
 orbs:
-  tools: gotest/tools@0.0.13
+  tools: gotest/tools@0.0.14
 
 executors:
   build:
     docker:
-      - image: circleci/golang:1.16
+      - image: cimg/go:1.18.8
```

Comment:
While we are here: Should the version be bumped to 1.19?

Reply:
sure, done